src/instruction/__tests__/utils.ts

```diff
@@ -0,0 +1,61 @@
+import { convertTypeToType } from "../utils";
+import {describe, expect, it} from "@jest/globals";
+
+describe("Type conversion with convertTypeToType", () => {
```

Comment:
Can you please add a test for `(arg0: any) => any` to match the current functionality still works?

Reply:
@long-banana Do you mean something like this?
https://github.com/raindrop-studios/sol-kit/pull/11/commits/2c75d8a5d0c49365ff39a4a2a7e324d9bbcf97a7

Reply:
yup. perfect!